sdk/executor/src/main/java/com/mesosphere/sdk/executor/CustomExecutor.java

```diff
@@ -192,61 +189,20 @@ public void shutdown(ExecutorDriver driver) {
         for (Map.Entry<Protos.TaskID, LaunchedTask> entry : launchedTasks.entrySet()) {
             final Protos.TaskID taskId = entry.getKey();
             try {
-                LOGGER.info("Stopping task as part of executor shutdown: {}", taskId);
+                LOGGER.info("Stopping task as part of executor shutdown: {}", TextFormat.shortDebugString(taskId));
```

Comment:
Maybe `taskId.getValue()`? Same info but without unnecessary brackets and field names, I assume.